local/compose/convergence.go

```diff
@@ -356,13 +354,15 @@ func (s *composeService) isServiceHealthy(ctx context.Context, project *types.Pr
 			return false, fmt.Errorf("container for service %q has no healthcheck configured", service)
 		}
 		switch container.State.Health.Status {
-		case "starting":
+		case moby.Starting:
 			return false, nil
-		case "unhealthy":
+		case moby.Unhealthy:
 			return false, nil
+		case moby.Healthy:
```

Comment:
nit: If we only care about healthy, we could make this a simple if or a switch ... default